[2025-09-06T21:37:54.408+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-06T21:37:54.446+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_model_dag.extract_features manual__2025-09-06T20:15:46.988039+00:00 [queued]>
[2025-09-06T21:37:54.468+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_model_dag.extract_features manual__2025-09-06T20:15:46.988039+00:00 [queued]>
[2025-09-06T21:37:54.469+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-06T21:37:54.498+0000] {taskinstance.py:2330} INFO - Executing <Task(SimpleHttpOperator): extract_features> on 2025-09-06 20:15:46.988039+00:00
[2025-09-06T21:37:54.513+0000] {standard_task_runner.py:63} INFO - Started process 275 to run task
[2025-09-06T21:37:54.526+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'train_model_dag', 'extract_features', 'manual__2025-09-06T20:15:46.988039+00:00', '--job-id', '291', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmpa0jnpoop']
[2025-09-06T21:37:54.531+0000] {standard_task_runner.py:91} INFO - Job 291: Subtask extract_features
[2025-09-06T21:37:54.635+0000] {task_command.py:426} INFO - Running <TaskInstance: train_model_dag.extract_features manual__2025-09-06T20:15:46.988039+00:00 [running]> on host fd946087d8e2
[2025-09-06T21:37:54.885+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_model_dag' AIRFLOW_CTX_TASK_ID='extract_features' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T20:15:46.988039+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T20:15:46.988039+00:00'
[2025-09-06T21:37:54.887+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-06T21:37:54.918+0000] {http.py:169} INFO - Calling HTTP method
[2025-09-06T21:37:54.921+0000] {base.py:84} INFO - Using connection ID 'preprocess_api' for task execution.
[2025-09-06T21:37:54.926+0000] {base.py:84} INFO - Using connection ID 'preprocess_api' for task execution.
[2025-09-07T07:36:36.838+0000] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-09-07T07:36:37.084+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-09-07T07:36:37.122+0000] {process_utils.py:132} INFO - Sending 15 to group 275. PIDs of all processes in the group: [275]
[2025-09-07T07:36:37.144+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 275
[2025-09-07T07:36:37.257+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-09-07T07:36:37.430+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-07T07:36:38.006+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=275, status='terminated', exitcode=0, started='21:37:54') (275) terminated with exit code 0
